tap_list_providers/parsers/untappd.py

```diff
@@ -220,15 +220,29 @@ def parse_style(self, style):
         name = style.strip()
         return Style.objects.get_or_create(name=style)[0]
 
-    def parse_size(self, size):
-        if size == "1/6 Barrel":
-            return 660
-        elif size == "1/4 Barrel":
-            return 996
-        elif size == "1/2 Barrel":
-            return 1980
-        else:
-            return int(size.split("oz")[0])
+    def parse_size(self, size: str) -> int:
+        custom_sizes = {
+            "1/6 barrel": 660,
+            "1/4 barrel": 996,
+            "1/2 barrel": 1980,
+            "flight": 5,
```

Comment:
Talking to Derrell at Chattahoochee, his "flight" size is actually 6 oz. Untappd has the serving size "flight" on a list, so I don't know if Untappd has a standard that says it's 5 oz., but Chattahoochee at least serves 6 oz. for that size.

Reply:
Nope, that was a guess on my part. We have venues that have 4 oz, 5 oz, and 6 oz tasters, so I split the difference. Since they're using 6 oz, I'll go with that.